### ISSUE TITLE
PM's - sa=search - broken link button, no labels.

### ISSUE THREAD

**Issue by StealthWombat** [elkarte/Elkarte#601]
The "Hide advanced options" button doesn't work. Methinks js is borked.

Also, the inputs really should have their associated text in label tags for better a11y.

A few minor layout tweaks wouldn't hurt either.
